src/effects/effectsmanager.cpp

```diff
@@ -32,6 +34,15 @@ EffectsManager::EffectsManager(QObject* pParent, UserSettingsPointer pConfig)
 
     m_pNumEffectsAvailable = new ControlObject(ConfigKey("[Master]", "num_effectsavailable"));
     m_pNumEffectsAvailable->setReadOnly();
+
+    ChannelHandleAndGroup master = ChannelHandleAndGroup(
+          m_pChannelHandleFactory->getOrCreateHandle("[Master]"), "[Master]");
+    registerInputChannel(master);
+    registerOutputChannel(master);
```

Comment:
I'll take another look at the initialization of the ChannelHandles and try to make it more self-contained and consistent. I may have overcomplicated it in the process of working on this and it can be simplified now.